app/helpers/payment_details_helper.rb

```diff
@@ -19,33 +22,37 @@ def link_to_payment_details(text, options = {})
   end
 
   def payment_details_path(merchant_account = site_account, url_params = {})
-    return "" if ["bogus" ,""].include?(merchant_account.payment_gateway_type.to_s)
-    named_route = [:admin, :account, merchant_account.payment_gateway_type]
+    payment_gateway_type = merchant_account.payment_gateway_type
+    return "" if ["bogus",""].include?(payment_gateway_type.to_s)
```

Comment:
We should favour single quotes for strings without interpolation. Also, we could remove this `.to_s` with
```suggestion
    return '' if [:bogus, nil].include?(payment_gateway_type)
```